SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialMultiWorkerSettings.h

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
```

Comment:
nit: classes in EngineClasses *should* be for spatial classes that extend default engine functionality (ie. spatialgameinstance, spatialnetconnection etc.).